### ISSUE TITLE
Add canonical URL by default

### ISSUE THREAD

**Issue by PJUllrich** [johantell/metatags#152]
Hey hey, it would be nice if the `Metatags.print_tags/1` call would always print the canonical URL, even if it isn't specified.

I have to always add this code in addition to using Metatags and it would be nice if these would be combined:

```elixir
    <meta rel="canonical" href={Phoenix.Controller.current_url(@conn, %{})} />
    <%= Metatags.print_tags(@conn) %>
```

**Comment by johantell** [comment]
I think that makes sense 👍 I'll try to implement that once I have some time on my hands. :)

**Comment by johantell** [comment]
@PJUllrich I just released v0.5.0 which includes this suggestion 👍
